pnparser/templates/border.py

```diff
@@ -0,0 +1,59 @@
+from typing import TYPE_CHECKING
+from .common import *
+
+if TYPE_CHECKING:
+    from .pnb import Pnb
+    from .pbc import Pbc
+    from .section import Section
+    from .splash import SplashTableEntry
+
+
+TEMPLATE: str = "{{{{pnbh|context='''{}'''<br>\n"
+
+
+class Border:
+    """class=\"content-border\""""
```

Comment:
im semi confused by this docstring? can you add a bit more info?

Reply:
Oh sorry, that only made sense on my head. That is how Riot separates each box on they post:
![image](https://user-images.githubusercontent.com/28227097/111363984-76f21300-866f-11eb-9dee-5ae6f424843f.png)

Reply:
Ah makes sense, yeah just add a couple words noting that it's the border class to the docstring then 👍